.ci/pipeline_integration_test

```diff
@@ -18,7 +19,7 @@ declare CONTROL_KUBECONFIG=${SOURCE_PATH}/dev/control_kubeconfig.yaml
 declare TARGET_KUBECONFIG=${SOURCE_PATH}/dev/target_kubeconfig.yaml
 declare TEST_SHOOT=az-ci-it
 declare TEST_CREDS_SECRET=test-mc-secret
-declare TARGET_RESOURCE_GROUP="shoot--mcm-ci--$TEST_SHOOT"
+declare TARGET_CLUSTER_NAME="shoot--mcm-ci--$TEST_SHOOT"
```

Comment:
it still exports `TARGET_RESOURCE_GROUP`. change it to `TARGET_CLUSTER_NAME` and pipeline will pass

Reply:
yeah, I saw this too. this script is a mess. It could use the `set +o allexport` to avoid hunting for stuff like this.